src/components/ciConfig/CIConfig.tsx

```diff
@@ -124,40 +126,65 @@ function Form({ dockerRegistries, sourceConfig, ciConfig, reload, appId }) {
             },
         },
         onValidation,
-    );
-    const [args, setArgs] = useState([]);
-    const [loading, setLoading] = useState(false);
-
+    )
+    const [args, setArgs] = useState([])
+    const [loading, setLoading] = useState(false)
+    const [platformError, setPlatformError] = useState<string>('')
+    const targetPlatformList: OptionType[] = [
+        { label: 'linux/arm64', value: 'linux/arm64' },
+        { label: 'linux/amd64', value: 'linux/amd64' },
+        { label: 'linux/arm/v7', value: 'linux/arm/v7' },
+    ]
+    let _selctedPlatforms = []
+    if (ciConfig && ciConfig.dockerBuildConfig && ciConfig.dockerBuildConfig.targetPlatform) {
```

Comment:
We can use ciConfig?.dockerBuildConfig instead

Reply:
Done